.github/update_conda_lock.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+
+from os import environ
+from subprocess import run, DEVNULL, PIPE
+from sys import exit
+from re import match, search
+
+def _run(cmd_string, multiword_last_arg='', return_stdout=False, **kwargs):
+    cmd = cmd_string.split() + ([multiword_last_arg]
+            if multiword_last_arg else [])
+    if return_stdout:
+        return run(cmd, check=True, encoding='utf-8', stdout=PIPE,
+                **kwargs).stdout
+    else:
+        return run(cmd, check=True, **kwargs)
+
+def _get_env(env_name, required=True):
+    if env_name not in environ:
+        if required:
+            print('ERROR: Required environment variable not found: ' + env_name + '!')
+        return None
+    env_var = environ[env_name]
+    print('* ' + env_name + ': ' + env_var)
```

Comment:
Should this print be behind a verbose flag?

Reply:
I'd rather have those seen by default in the workflow's log so I don't think this should be behind such flag.